src/main/java/org/betonquest/betonquest/quest/event/drop/DropEventFactory.java

```diff
@@ -0,0 +1,84 @@
+package org.betonquest.betonquest.quest.event.drop;
+
+import org.betonquest.betonquest.Instruction;
+import org.betonquest.betonquest.Instruction.Item;
+import org.betonquest.betonquest.api.common.function.Selector;
+import org.betonquest.betonquest.api.common.function.Selectors;
+import org.betonquest.betonquest.api.quest.event.Event;
+import org.betonquest.betonquest.api.quest.event.EventFactory;
+import org.betonquest.betonquest.api.quest.event.StaticEvent;
+import org.betonquest.betonquest.api.quest.event.StaticEventFactory;
+import org.betonquest.betonquest.exceptions.InstructionParseException;
+import org.betonquest.betonquest.quest.event.NullStaticEventAdapter;
+import org.betonquest.betonquest.quest.event.OnlineProfileGroupStaticEventAdapter;
+import org.betonquest.betonquest.quest.event.PrimaryServerThreadEvent;
+import org.betonquest.betonquest.utils.PlayerConverter;
+import org.bukkit.Location;
+import org.bukkit.Server;
+import org.bukkit.entity.Player;
+import org.bukkit.plugin.Plugin;
+import org.bukkit.scheduler.BukkitScheduler;
+
+/**
+ * Factory to create {@link Event}s that drop items from instructions.
+ */
+public class DropEventFactory implements EventFactory, StaticEventFactory {
+    /**
+     * Server to use for syncing to the primary server thread.
+     */
+    private final Server server;
+
+    /**
+     * Scheduler to use for syncing to the primary server thread.
+     */
+    private final BukkitScheduler scheduler;
+
+    /**
+     * Plugin to use for syncing to the primary server thread.
+     */
+    private final Plugin plugin;
+
+    /**
+     * Create the drop event factory.
```

Comment:
For example:
```suggestion
     * Creates the drop event factory.
```

Reply:
Well, we've got it mixed already and I personally prefer the style of documentation that could replace the method call (kind of) instead of the "This does something" (even if the this is omitted). But I can see that the majority is using the later style, so here's my commit adding a bunch of `s`.

Reply:
Thanks!